backend/app/api/routes/scripts.py

```
@@ -115,19 +86,16 @@ async def parse_script(
-        
-        # Return result
+        # Return result with empty tree (frontend handles parsing)
         result = {
             "script_id": script_id,
             "filename": file.filename,
-            "tree": node_to_dict(parsed_tree)
+            "tree": {}
         }
         
         return result
     except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)